jwql/utils/engineering_database.py

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/env python
+"""Module to interface the JWST Engineering Database.
+
+This module provides ``jwql`` with functions to interface and query the
+ JWST DMS Engineering Database.
+
+Authors
+-------
+
+    - Johannes Sahlmann
+
+Use
+---
+
+    This module can be imported and used with
+
+    ::
+
+        from jwql.engineering_database import query_single_mnemonic
+        query_single_mnemonic(mnemonic_identifier, start_time, end_time)
+
+    Required arguments:
+
+    ``mnemonic_identifier`` - String representation of a mnemonic name.
+    ``start_time`` - astropy.time.Time instance
+    ``end_time`` - astropy.time.Time instance
+
+Notes
+-----
+    This module is built on top of astroquery.mast and uses JWST-specific MAST services.
+    The user has to provide a valid MAST authentication token.
+
```

Comment:
@Johannes-Sahlmann It might be useful to add a `References` section here that points to any existing documentation on the EDB, and perhaps a link to the MAST portal for EDB.

Reply:
I added a link to the web portal.